scripts/fast-reboot

```diff
@@ -502,6 +505,11 @@ if [ -x /sbin/hwclock ]; then
     /sbin/hwclock -w || /bin/true
 fi
 
+if [ -x ${DEVPATH}/${PLATFORM}/${PLATFORM_PLUGIN} ]; then
+    VERBOSE=yes debug "Running ${PLATFORM} specific plugin..."
```

Comment:
Can you remove "VERBOSE=yes"? We would like to keep console quiet in normal execution.